var/spack/repos/builtin/packages/hwloc/package.py

```diff
@@ -53,3 +53,12 @@ class Hwloc(AutotoolsPackage):
 
     def url_for_version(self, version):
         return "http://www.open-mpi.org/software/hwloc/v%s/downloads/hwloc-%s.tar.gz" % (version.up_to(2), version)
+
+    def install(self, spec, prefix):
+        # Disable OpenCL, since hwloc might pick up an OpenCL library
+        # at build time that is then not found at run time
+        configure("--prefix=%s" % prefix,
+                  "--disable-opencl")
```

Comment:
This package uses `AutotoolsPackage` now, so you'll have to update things. Basically, delete this install method and replace it with:
```python
def configure_args(self):
    # Disable OpenCL, since hwloc might pick up an OpenCL library
    # at build time that is then not found at run time
    return ['--disable-opencl']
```